config.js

```diff
@@ -0,0 +1,15 @@
+require('dotenv').config()
+
+const { Pool } = require( 'pg' )
+const isProduction = process.env.NODE_ENV === 'production';
+
+const connectionString = `postgresql://${process.env.DB_USER}:${process.env.DB_PASSWORD}@${process.env.DB_HOST}:${process.env.DB_PORT}/${process.env.DB_DATABASE}`;
+
+const pool = new Pool({
```

Comment:
Nice work setting up the dynamic string.  I've never used `Pool` before but it looks like you got it running successfully.  Cheers!